content/authors/yanhongxu/_index.md

```diff
@@ -0,0 +1,41 @@
+---
+# Display name
+title: Yanhong Xu
+
+# Username (this should match your github username)
+authors:
+- yanhongxu520
+
+superuser: false
+
+# Role/position e.g. Graduate student
+role: Postdoc Associate
+
+# Organization membership
+Organizations/Affiliations:
+organizations:
+- name: University of Calgary
+  url:http://www.ucalgary.ca
```

Comment:
Can you add a space after the colon here? It should say
```
  url: http://www.ucalgary.ca
```
NOT
```
  url:http://www.ucalgary.ca
```

Reply:
Thanks, I just made a new commit.